data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,461 @@
+		has "event: hr: exploration period"
+	on offer
+		set "hr: media retraction"
+		event "hr: media retraction reaches hai" 25
 	# Sync reputations for aliases of governments:
 		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
 		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
-		# Make sure the new governments start with appropriate rep:
+	# Make sure the new governments start with appropriate rep:
 		"reputation: Elenctic Commune" = 1000
+
+event "hr: media retraction reaches hai"
+
+
+mission "Hai Reveal: Media Retraction (Human)"
+	landing
+	invisible
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+		attributes "spaceport"
+	to offer
+		has "hr: media retraction"
+		not "Hai Reveal: Media Retraction (Hai): offered"
+	on offer
 		conversation
 			`News report that says that the guy who showed the video of the Hai ships five months prior was just a viral marketing stunt for an upcoming movie.`
 				decline
 
 
+mission "Hai Reveal: Media Retraction (Hai)"
+	landing
+	invisible
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+		attributes "spaceport"
+	to offer
+		has "event: hr: media retraction reaches hai"
+	on offer
+		event "hr: pirates respond to leaks" 1
+		event "hr: concern period" 35
+		conversation
+			`News report that says that the guy who showed the video of the Hai ships five months prior was just a viral marketing stunt for an upcoming movie. With additional advisory warning for Hai merchants.`
+				decline
+
 
-event "pirates respond to leaks"
+event "hr: pirates respond to leaks"
 	# Add pirate spawns to Waypoint and Heia Due
 	system "Waypoint"
 		government "Pirate"
-		add fleet "Large Northern Pirates" 3000
-		add fleet "Small Northern Pirates" 1000
-		add fleet "Large Core Pirates" 3000
-		add fleet "Small Core Pirates" 1000
+		add fleet "Large Northern Pirates" 5000
+		add fleet "Small Northern Pirates" 2500
+		add fleet "Large Core Pirates" 6000
+		add fleet "Small Core Pirates" 3000
 		remove fleet "Small Hai"
 		remove fleet "Large Hai"
 	system "Heia Due"
-		add fleet "Marauder fleet IX" 3000
-		add fleet "Large Northern Pirates" 3000
-		add fleet "Large Core Pirates" 3000
-		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
+		add fleet "Marauder fleet IX" 15000
+		add fleet "Large Northern Pirates" 10000
+		add fleet "Large Core Pirates" 12000
+		add fleet "Anti-Pirate Hai (Wormhole Travel)" 2000
+
+event "hr: concern period"
+
+
+
+mission "Hai Reveal Concern Period: Humans Going Home"
+	landing
+	invisible
+	source
+		government "Hai"
+		not attributes "uninhabited" "retirement"
+	destination
+		attributes "hai: human presence"
+	to offer
+		has "event: hr: pirates respond to leaks"
+		random < 10
+	on offer
+		conversation
+			`Humans with business on worlds that have very little human presence are moving or returning to homes where there are more humans. They're not coping with the suspicion directed at them.`
```

Comment:
~~you forgot the passengers~~
```suggestion
			`As you land you are hailed by the local Hai authorities, "<last>, we have a situation developing between the humans and Hai. Tensions are rising and we fear it may worsen if we do not act soon." Another Hai interrupts them, "This is why we want your help, captain, we fear that asking Hai captains could easily degenerate. We think these <passengers> humans would be safer on <destination>."`
			`	You go to the spaceport to meet with these humans, and speak to what appears to the leader of the group. Judging by their expensive clothing and somewhat impatient attitude they must be business men, or perhaps even diplomats.`
			choice
				`	(Invite them to my ship.)`
					accept
				`	"What seems to be the problem?"`
			`	"Ah, I trust you are the Captain that has been allocated to help us. I am Brin." You introduce yourself in response, and they continue, "We are conducting business here, as we have been for some time in fact, but with the recent events suspicion has arisen. Some of them believe we will abduct them when we ask to move for more private locations to conduct contracts!" They say with a chuckle. "As you can imagine, business is suffering as a result, and we will be more than happy to give you a just compensation of <payment> for this transport. We were hoping that the small amount of humans was a good business opportunity, which was only true until recently. Indeed, my colleagues and I will return to a location where we used to do business in the past, but you understand we cannot just ask any captain to take us, even if we are willing to make do without proper accommodations due to the relative urgency of the situation."`
```